server/__init__.py

```diff
@@ -180,6 +194,12 @@ def create_app():
   register_routes_common(app)
   if cfg.CUSTOM:
     register_routes_custom_dc(app)
+  if cfg.ENV_NAME == 'STANFORD':
+    register_routes_stanford_dc(app)
```

Comment:
wonder if we should also register_routes_stanford_dc for local or autopush (like the original gating we had on the APIs)? what do you think @beets?

Reply:
yes, autopush for now would be nice, though the data might deviate at some point.

Reply:
Added autopush.